util/perl/TLSProxy/Proxy.pm

```diff
@@ -243,16 +244,19 @@ sub start
     if ($self->debug) {
         print STDERR "Server command: $execcmd\n";
     }
+    my $sin = undef;
+    my $sout = undef;
+    if ("$^O" eq "MSWin32") {
+        $pid = IPC::Open2::open2($sout, $sin, $execcmd) or die "Failed to $execcmd: $!\n";
+    } else {
+        $pid = IPC::Open3::open3($sin, $sout, undef, $execcmd) or die "Failed to $execcmd: $!\n";
```

Comment:
Shouldn't there be a corresponding `use IPC::Open3` somewhere?

Reply:
I don't know. the master branch is happy without `use IPC::Open3`

Reply:
Oh, it seems like IPC::Open3 is a standard module, i.e. "used" by default